### PR TITLE
Fix code doc comment that referenced a private member.

### PR DESCRIPTION
**Pull request by jacob314**

### CODE REVIEW

**Review by Hixie — APPROVED** [review]
LGTM

thanks!

File: packages/flutter/lib/src/widgets/widget_inspector.dart [hunk 1]
```diff
@@ -3310,8 +3310,7 @@ class InspectorSerializationDelegate implements DiagnosticsSerializationDelegate
   ///   String groupName,
   ///   int subtreeDepth,
   /// ) {
-  ///   return _nodeToJson(
-  ///     root,
+  ///   return root.toJsonMap(
   ///     InspectorSerializationDelegate(
   ///       groupName: groupName,
   ///       summaryTree: false,
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/widget_inspector.dart:3316]: the other thing that's weird about this sample code is the `service: this` bit. Are most people who call this going to be saying `this`? (Context is I'm trying to set up code analysis and I'm not sure how to make this pass.)

Comment (jacob314) [reply packages/flutter/lib/src/widgets/widget_inspector.dart:3316]: Done. Replaced with WidgetInspectorService.instance
Let me know if anything else is needed to make the code analysis pass.

Comment (Hixie) [reply packages/flutter/lib/src/widgets/widget_inspector.dart:3316]: awesome, looks great, thanks!

**Comment by jacob314** [comment]
Closing as we can just delete this code sample.

**Pull request closed without merging**